test/corpus/literals.txt

```diff
@@ -238,6 +243,8 @@ f"a {b}}}"
 f"a {{{b}"
 f"a {{b}}"
 f"a {{{b}}}"
+f"{c,}"
+f"{yield d}
```

Comment:
Is the quotation intentionally unmatched?

Reply:
Nope. This was a mistake on my part. I must have accidentally left it off, and, well, just seen the `ERROR` in the output. That's one pitfall of doing it this way.

Thanks for catching this!